packages/hooks/src/useReactive/__tests__/index.test.tsx

```diff
@@ -2,9 +2,10 @@ import { fireEvent, render } from '@testing-library/react';
 import React from 'react';
 import { act } from 'react-dom/test-utils';
 import useReactive from '../';
+import { renderHook } from '@testing-library/react-hooks';
```

Comment:
```suggestion
import { renderHook, act } from '@testing-library/react';
```

合下 master，`@testing-library/react-hooks` 不支持 react 18 了

Reply:
这个 act 也需要重新从 `@testing-library/react` 导出吗